include/benchmark/benchmark_api.h

```diff
@@ -616,9 +628,25 @@ class Benchmark {
   Benchmark(Benchmark const&);
   void SetName(const char* name);
 
+  int ArgsCnt() const;
+
+  static void AddRange(std::vector<int>* dst, int lo, int hi, int mult);
+
 private:
   friend class BenchmarkFamilies;
-  BenchmarkImp* imp_;
```

Comment:
Sorry! I figured since we already use STL types in `State` there was no need to keep PIMPL'ing benchmark. Mostly it was just the cost of the double dispatch that I wanted to remove.